tool/scrape_mdn.dart

```diff
@@ -118,6 +120,33 @@ final RegExp _xrefRegex =
     RegExp(r'''{{\s*(\w+)\(([\w\., \n/\*"'\(\)]+)\)\s*}}''');
 final RegExp _mustacheRegex = RegExp(r'''{{\s*([\S ]+?)\s*}}''');
 
+String? findTitle(String content) {
+  // Look for 'title: AbortController'.
+
+  for (var line in content.split('\n')) {
+    line = line.trim();
+
+    // early exit
+    if (line.isEmpty) return null;
+
+    if (line.contains(':')) {
+      final index = line.indexOf(':');
+      final key = line.substring(0, index).trim();
+
+      if (key == 'title') {
+        var value = line.substring(index + 1).trim();
+        if (value.contains(' ')) {
```

Comment:
nit: alternatively, just do a split and get the first value in the list.

Reply:
👍 done